src/components/Book.jsx

```diff
@@ -15,6 +15,7 @@ const Book = ({
   return (
     <div key={id} className="book-section-one">
       <div className="genre-section">
+        <small id="book-category">Action</small>
         <div className="book-title">
           <h2>{title}</h2>
         </div>
```

Comment:
- [ ] As required for the project to **EXACTLY** match the `Zeplin` design, kindly ensure that you use the same `fonts(e.g.: family, weight, size,...)`, `color`, `spacing`,.... like the one used on the template
### Your design
![Screenshot from 2023-06-23 02-25-35](https://github.com/Katended/book-store/assets/83042742/b73726c7-9015-42bd-9738-acb41ebedbf0)

### Template design
![Screenshot from 2023-06-23 02-33-30](https://github.com/Katended/book-store/assets/83042742/9fa5dd50-b69d-4b9b-b874-2f09ed5fedf9)